tests/yast2_gui/yast2_control_center.pm

```diff
@@ -273,6 +291,12 @@ sub run {
     if (is_sle && sle_version_at_least '15') {
         #see bsc#1062331, sound is not added to the yast2 pattern
         ensure_installed('in yast2-sound');
+        #kdump is disabled by default, so ensure that it's installed
+        ensure_installed('in yast2-kdump');
```

Comment:
how is "yast2-kdump" related to the comment that "kdump is disabled by default"? Btw, it shouldn't be disabled by default because as far as I remember at least on a default SLES installation it will enable kdump during installation, isn't it?

Reply:
kdump is disabled in installation by default, hence nothing is installed and this is expected. In the qcow inage used here it's disabled and we lack defined flow how to be sure if it's there or not in given image.

Reply:
right, it's disabled by default. I also checked [SLE 12-SP3](https://openqa.suse.de/tests/1059658#step/installation_overview/1), nevermind